lib/simulator-xcode-6.js

```diff
@@ -859,6 +859,22 @@ class SimulatorXcode6 extends EventEmitter {
   }
 
   /**
+   * Generate Apple script, which activates Simulator.
+   *
+   * @returns {string} the generated Apple Script snipped.
+   */
+  async generateWindowActivationScript () {
```

Comment:
Why make this `async`?

Reply:
Oh, I guess because a subclass needs it to be?

Reply:
yes, exactly